features/partners.feature

```diff
@@ -8,7 +8,8 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
   # Dit betekent dat in 'partners' zowel actuele partners zitten als ontbonden partners
   # Een proxy vertaalt dit naar de vorm Persoon, waarin sommige gegevens in bewerkte vorm worden opgenomen en waarin informatievragen kunnen zitten
   # Een voorbeeld van een informatievraag is naam.aanschrijfwijze, waarin op basis van de aanduiding naamgebruik, de naam van de persoon en de naam van de (ex)partner een aanschrijfnaam wordt samengesteld
-  # De proxy verwijdert de partners met datum ontbinding die geleverd zijn in de GbaPersoon
+  # Als er een actuele partner is (huwelijk / partnerschap is niet ontbonden) verwijdert de proxy de partners met datum ontbinding die geleverd zijn in de GbaPersoon
+  # Als er alleen ontbonden huwelijken/partnerschappen zijn levert de proxy alleen de ex-partner met de meest recente datum_ontbinding
```

Comment:
Wat is de reden van de '_'? Hierdoor is dit niet hetzelfde als de property naam of functionele naam.

Reply:
Geen idee.. Ik pas hem aan